### ISSUE TITLE
Updated Responsive Website and Logo Design Proposal

### ISSUE THREAD

**Issue by wick377** [qbittorrent/qBittorrent-website#155]
![qbittorrent mock](https://user-images.githubusercontent.com/62957338/101672447-b230e480-3a77-11eb-885c-e81ca7ac484a.png)
I am creating a modern and catchy look for qbittorrent website.
The design is in Bootstrap 4 and is completely responsive.
I have only designed the landing page for now. I will add more if approved.
You can check it out at [https://wick377.github.io/qbittorrent/](https://wick377.github.io/qbittorrent/)

**Comment by FranciscoPombal** [comment]
A for effort. Personally I don't like it though. Too much color, too much going on, too low information density and too much bloat (the transferred size is 10x the current version, with a ton of JavaScript, a major regression). At least it seems to display seamlessly on mobile. But then again, so does http://bettermotherfuckingwebsite.com/ and https://ninja-build.org/, at about 100x or less the size (and the current one is also not too bad on mobile).

As you might have guessed, yes, I'm biased toward minimalist and information density in websites. Browsing https://sourcehut.org is 1000x better experience for me than GitHub, for instance. This is more the direction I'd like the website to take, but hey, at the end of the day, I don't decide these things alone.

**Comment by wick377** [comment]
![qbittorrent mock](https://user-images.githubusercontent.com/62957338/101679700-80bd1680-3a81-11eb-97c3-17d8ae1450cf.png)
Thank you for checking it out.
I agree with you on the Javascript matter and I have modified the website a little bit, to show how it can be made more simple.
But I have to disagree with you on information density.
The current website has a dated design and shows too much data on a single screen. My design is more minimal and shows only the required information to a visitor. They can scroll if they want to see more.
The qbittorrent logo on the right is just to show you the logo. It will be replaced by a snapshot of the software.

**Comment by jagannatharjun** [comment]
really liked the designs maybe you can play around with qbittorrent colors for better background colors, it seems too much vibrant
is it your own design or based on some template?

**Comment by wick377** [comment]
I have tried to get the colors closer to qbittorrent color scheme. The current website is monotone and very dull that is why I have added some colors to make it a bit catchy and modern. For the right side, I couldn't decide any color that would look good so I went with orange. I am open to suggestions for that. Besides that I would really like to go with this color scheme.
I have adopted styles from multiple themes I own and UX is designed in Adobe XD.
I would really appreciate if you could make me a member of qbittorrent so we can work together on the project and make open-source modern.

**Comment by Poopooracoocoo** [comment]
Why hide all of the menu items into a menu? Even on smaller sized windows (my size 1440 x 900), there's a menu and your logo takes up the full width, pushing the download button, some of the only information on that page, below the fold. 

The menu doesn't need to be hidden! It can overflow onto another line or scroll horizontally. I hate using websites with hamburger menus. 

I like the use of Barlow! The condensed-ness reminds me of Bahnschrift. Qbittorrent doesn't really have its own personality as its (default) interface just adapts to the operating system's default styles so it's interesting to see this. The website is the only place for Qbittorrent to show some personality. 

I think that the background could use more blue. The purple, pink and orange is rather Instagrammy lol. You don't see much of the blue and green, especially without scrolling down (at least on screens such as my laptop).

I like that little banner that tells you what the latest version is. The donate button is also something that the website needs!

RE content, I agree that the current website has a lot of unnecessary information for a lot of users. Your website could perhaps do with some screenshots and a brief outline of some features

btw what's the JS for? i'm guessing that most of the stuff is unused, but that's beside the point as it may be a WIP? 

note: i'm just a random user

**Comment by wick377** [comment]
@Poopooracoocoo Thanks for the review.
The application is aimed at Windows users which is why on smaller devices I am using Hamburger to avoid unnecessary clutter.

The logo is temporary to show the design and will be replaced by a snapshot of the software.

As for the colors, I couldn't find a suitable color to replace orange. I am open to suggestions for that. The purple color matches the qbittorrent color scheme and looks good. I have given it a touch of blue but purple looks lots better. Orange definitly needs replacement.

JS will be removed in the final version. This is just a mockup and I have copied styles from multiple templates I own.

**I am very looking forward to heads up from the developers to start working on the new website**

**Comment by Poopooracoocoo** [comment]
>The application is aimed at Windows users which is why on smaller devices I am using Hamburger to avoid unnecessary clutter.

Most of my apps are 1440 x 900 when not maximised on my laptop with a 1080p display. I use an app called Sizer 😛. There shouldn't be a hamburger menu at that size. GitHub has a hamburger menu when the width is smaller than 1152. Your website gets a hamburger menu when the width is smaller than 1800. Arguably, GitHub has simpler navigation despite being a significantly more complex site, but idk if having a hamburger menu at all is a good idea _because_ it's not as complex as sites such as GitHub. 

>This is just a mockup and I have copied styles from multiple templates I own.

ah understood.

**Comment by wick377** [comment]
Oh I see that now @Poopooracoocoo 
There will be lots of chances based on reviews like yours. As I said before this is just a sample to show hoe everything is gonna be. I have copied the nav items from qbittorrent website for now. In the final version, you can scroll down to see more content like features and about info, eliminating the need for extra nav items..

I can see that this design does not elaborate my idea enough. Sorry @FranciscoPombal 
The landing page will be scrollable. I have shown only necessary information here in the photo but when the page is scrolled, you'll be able to see features section and about section.

**Comment by symonxdd** [comment]
@wick377 Personally, from an IT student's standpoint, I absolutely love your design!
Perhaps it has a little bit too much (bright) colors going on, but that's the only thing.
I'm a design enthusiast and your design is definitely a million times better than the current one, +1 from me 👍

**Comment by danfiscus** [comment]
It looks like this thread has been abandoned, but I really want to give my two cents here. I think this is a much much better website than the current one. A slick design is really important these days. Potential users will be turned off by a website that is clunky or appears outdated. The current site is badly in need of a modern look, it almost looks untrustworthy and GeoCities-esque. I think it's a damn shame that the maintainers of the site disagree on this.
@symonxdd is absolutely right, a landing page shouldn't contain all the relevant information about a product or service. It should briefly present the key features without overwhelming the reader. Technical details are important too, but that information is better suited for a features page or something.

**Comment by Poopooracoocoo** [comment]
@danfiscus Thoughts on PR #164? Do the graphics look more professional?

**Comment by wick377** [comment]
@danfiscus @symonxdd you are absolutely right! The landing page shouldn't contain all the details about the product.
An average user just wants the download button to access the product which is hard to find in the current design.
I personally found qbittorrent while reading reviews on other websites and didn't care about any details mentioned on app's website.
Besides a separate features page allows for more space and freedom to deliver the info in an easy to understand manner.

**Comment by wick377** [comment]
@Poopooracoocoo #164 doesn't really add anything to make the website look better. A few circles aren't gonna do anything.
It needs a complete redesign

**Comment by scratchmex** [comment]
bump? this looks amazing to me. Maybe the background can be changed to something "less" colorful and the download button can be highlighted more but the overall layout seems ok for me.
